source/_posts/vue-design-patterns.md

```diff
@@ -0,0 +1,982 @@
+---
+title: Vue Design Patterns
+contributor: Mauricio Matias C.
+avatar: mauricio-matias.jpeg
+description: Learn modern Vue 3 design patterns to make the best architectural decisions. Are you reinventing the wheel? 🧐.
+date: 2023-02-18
+tags: [vue, architecture, design-patterns]
+id: vue-design-patterns
+relatedPost: vue-security-best-practices
+bannerHeader: 'Are you reinventing the wheel? 🧐'
```

Comment:
@bartosz-io I'm not sure if contributors should add `bannerHeader` or `bannerSubheader` props

Reply:
I copied the `bannerHeader` style of `vue-security-best-practices.md`:

> bannerHeader: 'Is your Vue application REALLY secure? 🧐'